eng/scripts/Language-Settings.ps1

```diff
@@ -678,11 +678,17 @@ function Get-java-DocsMsMetadataForPackage($PackageInfo) {
   }
 }
 
-function Validate-java-DocMsPackages ($PackageInfo, $DocValidationImageId) 
-{
-  if (!(ValidatePackage $PackageInfo.Group $PackageInfo.Name $PackageInfo.Version $DocValidationImageId)) 
-  {
-    Write-Error "Package $($PackageInfo.Name) failed on validation" -ErrorAction Continue
+function Validate-java-DocMsPackages ($PackageInfo, $PackageInfos, $DocValidationImageId) {
+  # While eng/common/scripts/Update-DocsMsMetadata.ps1 is still passing a single packageInfo, process as a batch
+  if (!$PackageInfos) {
+    $PackageInfos =  @($PackageInfo)
+  }
+
+  foreach ($package in $PackageInfos) {
+    if (!(ValidatePackage $package.Group $package.Name $package.Version $DocValidationImageId)) {
```

Comment:
Based on my understanding, our docker is using `configuration_path` for batch work. I think it is better to pack all packages into configuration path and pass them into `ValidatePackage ` func. And change ValidatePackage accordingly.

Reply:
I have that work pending in another branch.   I was trying to keep this change a simple as possible to unblock my eng/common change.